src/Cli/dotnet/Parser.cs

```diff
@@ -281,6 +281,28 @@ private static void SetHelpCustomizations(HelpBuilder builder)
                 }
             }
 
+            public void additionalOption(HelpContext context)
```

Comment:
Let's rename this so it's more clear what this means.  Also, method and class names should usually be "PascalCased", ie the first letter of each word capitalized.


```suggestion
            public void WriteAdditionalMSBuildOptions(HelpContext context)
```